velox/functions/sparksql/tests/SparkCastExprTest.cpp

```diff
@@ -291,6 +291,46 @@ TEST_F(SparkCastExprTest, intToTimestamp) {
   testIntegralToTimestampCast<int32_t>();
 }
 
+TEST_F(SparkCastExprTest, doubleToTimestamp) {
+  testCast(
+      makeNullableFlatVector<double>({
+          0.0,
+          1727181032.0,
+          -1727181032.0,
```

Comment:
Please add the test that digit value is not 0 such as 12345678901234.786， and the value should be more than max float